src/libcore/marker.rs

```diff
@@ -135,6 +135,7 @@ pub trait Unsize<T: ?Sized> {
     // Empty.
 }
 
+
```

Comment:
Why this extra newline?

Reply:
That's a holdover from when `CoerceSized` was there and should be removed